vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -129,7 +129,7 @@ def call(Map pipelineParams) {
                                 region: '',
                                 versions: [],
                                 sub_tests: [],
-                                labels: ['master-weekly']
+                                labels: ['master-daily']
```

Comment:
@juliayakovlev FYI,

@radoslawcybulski as we discussed, this one is one every master build (which is move or less daily cadence)